schnorr_fun/src/musig.rs

```diff
@@ -124,78 +125,112 @@ pub struct KeyList {
     agg_key: Point<Normal>,
     /// The tweak on the aggregate key
     tweak: Scalar<Public, Zero>,
-    /// Whether this aggregate key needs negation.
-    accum_needs_negation: bool,
 }
 
-impl KeyList {
-    /// The `XOnly` aggregated key for the keylist.
-    pub fn agg_public_key(&self) -> XOnly {
-        self.agg_key.to_xonly()
+impl AggKey {
```

Comment:
Could rename `AggKey` to `MuSigKey` like we are planning with `FrostKey` which I quite like. But maybe this makes `Bip340MuSigKey` too long

With frost we will also have a `Bip340FrostKey` / `Bip340AggKey`

Reply:
Yeah let's apply the same thing to FROST and see what naming we like best. I think what you've suggested is right.